pillarbox-player/build.gradle.kts

```diff
@@ -15,6 +17,19 @@ android {
         buildConfig = true
     }
 
+    buildTypes {
+        debug {
+            buildConfigField("String", "VERSION_NAME", "\"debug\"")
+        }
+
+        release {
+            val latestTagHash = execCommand("git", "rev-list", "--tags", "--max-count=1")
+            val versionName = execCommand("git", "describe", "--tags", "$latestTagHash")
+
+            buildConfigField("String", "VERSION_NAME", "\"${versionName}\"")
+        }
+    }
+
```

Comment:
This job is already made by the CI, check build.gralde for analytics project

Reply:
Thanks for the pointer!
I've done the change in https://github.com/SRGSSR/pillarbox-android/pull/596/commits/84bf863bc057e89f82578b0efbb5c04b8b3a5947